extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -14,5 +14,12 @@
  * limitations under the License.
  */
 
- amp-sticky-ad {
- }
+amp-sticky-ad {
+  position: fixed !important;
+  text-align:center;
+  bottom: 0 !important;
+  left: 0;
+  z-index: 1;
+  width: 100%;
+  max-height: 100px !important;
```

Comment:
I tried to do that at first, but it seemed to me that css doesn't support min() function